packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -139,10 +164,10 @@ class SimControl {
     ]);
   }
 
-  Future<RunResult> install(String deviceId, String appPath) {
-    Future<RunResult> result;
+  Future<RunResult> install(String deviceId, String appPath) async {
+    RunResult result;
     try {
-      result = processUtils.run(
+      result = await _processUtils.run(
```

Comment:
The issue indicates that the user might be able to avoid the ProcessExceptions by fixing a plist file. Should the `throwToolExit` include that in the message?

Reply:
Done.